ecl/eclagent/eclgraph.cpp

```diff
@@ -927,6 +924,11 @@ void EclSubGraph::updateProgress(IStatisticGatherer &progress)
     }
     ForEachItemIn(i2, subgraphs)
         subgraphs.item(i2).updateProgress(progress);
+
+    Owned<IStatisticCollection> statsCollection = progress.getResult();
+    const cost_type costDiskAccess = statsCollection->aggregateStatistic(StCostFileAccess);
+    if (costDiskAccess)
+        progress.addStatistic(StCostFileAccess, costDiskAccess);
```

Comment:
the aggregated total should still be published at the global level after this change (unless I'm missing something),
but instead of it being calculated and set directly into global stats, it is calculated and set indirectly via the statsAggregator (inside updateProgress).

Reply:
Previously, this statistic wasn't picked up by the aggregator.  It needs to be in the subgraph scope.  However, the aggregator will add the statistic back in as a global statistic.